llvm/test/tools/llvm-readobj/ELF/dynamic-tags.test

```diff
@@ -138,6 +141,316 @@
 # GNU64-NEXT:  0x0000000076543210 (<unknown:>0x76543210) 0x5555666677778888
 # GNU64-NEXT:  0x0000000000000000 (NULL)                 0x0
 
+#      JSON64:"DynamicSection": [
```

Comment:
Nit: the other check patterns don't include a space after the colon, because they need to match with `--match-full-lines --strict-whitespace`. If you're not doing the same here, I'd add that space, to match the common style in most tests (and one on each other line too for consistency).

Reply:
done